doc/setup.md

```diff
@@ -14,14 +14,14 @@ For now we do not provide binaries for Linux or macOS. You can build the library
 
 ```bash
 sudo apt-get update
-sudo apt-get install build-essential cmake libfftw3-dev git
+bash ./.github/scripts/install_dependencies.sh
 ```
 ````
 
 ````{tab} macOS
 
 ```bash
-brew install cmake fftw
+brew install cmake gnu-tar bzip2 gzip fftw doxygen
 ```
```

Comment:
The changed dependencies are specific to doc which has a separate section below. If folks are not using ubuntu/debian that step won't work. Having the apt command their is fine since it tells you what the packages are without having to dig through that script